src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1633,9 +1571,12 @@ credentials:
 mysqlPassword = credentials.get('MY_MYSQL_PASSWORD')
 ----
 
+NOTE: The `credentials` <<_glossary_script_bindings,script binding>> can also be used in other <<_glossary_additional_task_scripts,scripts>> defined inside a <<_glossary_task,ProActive Task>>.
```

Comment:
Instead of `other scripts` we can use the term used before `additional scripts`.
We should also make the link cover `other scripts` or `additional scripts` (instead of just only `scripts`), depending on what you choose to do.

Reply:
ok make sense